src/features/keyframes/components/dopesheet-editor/easing-preset-thumbnail.css

```
@@ -0,0 +1,50 @@
+/**
+ * Square easing-preset thumbnails. Each tile shows a muted dot grid, a white
+ * easing curve, and an orange marker resting at the curve's start. On hover the
+ * marker rides the curve (via CSS motion path) to preview the timing. The
+ * animation is paused at rest, so a grid of ~25 thumbnails costs nothing until
+ * hovered — matching motion-preset-thumbnail.css.
+ */
+
+.ep-thumb {
+  --ep-grid: color-mix(in oklab, var(--muted-foreground) 42%, transparent);
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)